### PR TITLE
coin

### PR DESCRIPTION
**Pull request by Mijildinho**
idone

### CODE REVIEW

**Comment by Mijildinho** [comment]
Done.

**Comment by khatchad** [comment]
Do you have a grade for this in Blackboard? Did you enter your GitHub user name there?

**Comment by khatchad** [comment]
Closing. Please let me know if you didn't get a grade.

**Comment by Mijildinho** [comment]
did not get the grade

**Comment by khatchad** [comment]
@Mijildinho You got a grade, it's just 0 :). Let me check out your pull request now.

**Comment by khatchad** [comment]
Why are there two java files in your project? There should just be one.

File: src/coinss/Coinss.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        // TODO code application logic here
+        final int QUARTER_VALUE = 25;
+        final int DIME_VALUE = 10;
+        final int NICKEL_VALUE = 5;
+// TODO: Ask the user for D.
```
Comment (khatchad) [inline src/coinss/Coinss.java:24]: If it's done, you can erase the TODO.

File: src/coinss/Coinss.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        System.out.println("I now have " + dimeCount + " dimes.");
+        d %= DIME_VALUE;
+        System.out.println("I have " + d + " left to go.");
+// TODO: Calculate the number of nickels.
+        int nickelCount= d/ NICKEL_VALUE;
+        System.out.println("I now have " + nickelCount + " nickels.");
+        d %= NICKEL_VALUE;
+        System.out.println("I have " + d + " pennies left to go.");
+// What's left in d is the amount of pennies.
+// TODO: Print the number of coins in each category.
```
Comment (khatchad) [inline src/coinss/Coinss.java:45]: What happened to this part?

**Comment by khatchad** [comment]
Which file did you want me to grade?

**Comment by Mijildinho** [comment]
professor: i cant resubmitted through blackboard for your attention, therefore im commenting here to let you know that the one that should be graded in this lab should call "final version coin lab 2" thank you and see you in class

**Comment by khatchad** [comment]
You should be able to submit your GitHub username in Blackboard. I need that so that I can give your group a grade. What is the problem you are having?

**Comment by khatchad** [comment]
Closing. Please reopen if there is any trouble.

**Pull request closed without merging**